src/layout.cpp

```diff
@@ -15,84 +15,62 @@
 //*****************************************************************************
 #include "layout.hpp"
 
-#include <algorithm>
-#include <sstream>
-#include <vector>
-
-#include "stringutils.hpp"
-
 namespace ovms {
 
-LayoutConfiguration::LayoutConfiguration(const char* layout) :
-    LayoutConfiguration(std::string(layout)) {
+Layout::Layout() :
+    std::string() {
 }
```

Comment:
Wouldn't delcaring Layout() = default suffice in header?

Reply:
done